zap/src/main/installer/resources/Messages.properties

```diff
@@ -4,19 +4,19 @@ zap.installer.uninstalloldversion.text = An older version was detected. The olde
 zap.installer.uninstalloldversion.button = Uninstall
 zap.installer.uninstalloldversion.errormsg = Failed to uninstall older version. Continue with installation of newer version?
 
-zap.installer.cfu.screentitle = Check for Updates
-zap.installer.cfu.screensubtitle = Configure how ${compiler:sys.fullName} should check for updates
-zap.installer.cfu.title = Check for Updates on startup
-zap.installer.cfu.releases.title = ZAP Releases
-zap.installer.cfu.releases.autodownload = Automatically download new ZAP releases
-zap.installer.cfu.addonupdates.title = Add-on updates
-zap.installer.cfu.addonupdates.check = Check for updates to the add-ons you have installed
-zap.installer.cfu.addonupdates.autoinstall = Automatically install updates to the add-ons you have installed
-zap.installer.cfu.addonupdates.autoinstallscanners = Automatically install updates to the scanner rules you have installed
-zap.installer.cfu.newaddons.title = New Add-ons
-zap.installer.cfu.newaddons.reportrelease = Report new release status add-ons
-zap.installer.cfu.newaddons.reportbeta = Report new beta status add-ons
-zap.installer.cfu.newaddons.reportalpha = Report new alpha status add-ons
+zap.installer.cfu.screentitle = Check for Updates.
```

Comment:
Just discussed this PR via IRC. I think this is the last thing that needs tweaking.

This is a title for the screen, we feel it seems out of place with a period, so if you could revert this one that'd be great.

![image](https://user-images.githubusercontent.com/7570458/216368048-3a080336-d466-45ca-86bc-d203b95a2617.png)